snuba/clickhouse.py

```diff
@@ -375,6 +377,52 @@ def for_schema(self):
         return ', '.join(column.for_schema() for column in self.columns)
 
 
+class DatasetEnv(Enum):
+    PROD = 'prod'
+    DEV = 'dev'
+    TEST = 'test'
+
+
+class TableSchema(object):
+    """
+    Represents the full set of columns in a clickhouse table, this only contains
+    basic metadata for now. The code to generate the schema comes in a followup PR.
+    """
+    DATABASE = None
+
+    TABLE_TYPE_LOCAL = 'local'
+    TABLE_TYPE_DIST = 'dist'
+    TABLE_TYPE_QUERY = 'query'
+
+    TABLE_NAMES = {
+        TABLE_TYPE_LOCAL: {
+            DatasetEnv.PROD: None,
```

Comment:
If we will be able to avoid using different names between dev test and prod we can remove a lot of this. Right now it would be easy to remove the TEST table by auto generating the name but dev vs prod is more complicated because in dev we use the same table for local and distributed while in prod they are two different names.